packages/wonder-blocks-modal/components/one-column-modal.js

```diff
@@ -2,15 +2,17 @@
 import * as React from "react";
 import {StyleSheet} from "aphrodite";
 
-import {View, MediaLayoutWrapper} from "@khanacademy/wonder-blocks-core";
 import Color from "@khanacademy/wonder-blocks-color";
+import {
+    View,
+    MediaLayoutWrapper,
+    type MediaSize,
```

Comment:
Can you move this inline type out? It seems like it allows that in other places (and this runs counter to what we'd want to do with the Flow type checks, I think).

Reply:
I don't think there's a way to do this.  I can remove `import/no-duplicates`.

Reply:
I guess I don't actually need to remove it.  It might differentiate between type imports and regular imports.